app/util/autoversioning.py

```diff
@@ -20,18 +20,30 @@ def get_version():
     :return: The version of the application
     :rtype: str
     """
-    if getattr(sys, 'frozen', False):
-        return _get_frozen_package_version()  # frozen/packaged
 
-    return _calculate_source_version()  # unfrozen/running from source
+    # Initialize default version to '0.0.0'. So in case of failure the version will be shown as '0.0.0'.
+    version = '0.0.0'
```

Comment:
I've also been wondering if we should remove app/util/package_version.py from the repo completely. It seems like having a file already exist there with an invalid version adds more complexity than if we just wrote that file as needed. That would also simplify the code that backs up and restores the original package_version file.

Basically the code in that case would look like:
```python
def _get_frozen_package_version():
    try:
        from app.util import package_version
        return package_version.version
    except ImportError:
        return None


def _calculate_source_version():
    try:
        ...  # existing logic
    except subprocess.CalledProcessError:
        _calculated_version = None
    return _calculated_version


def get_version():
    return _get_frozen_package_version() or _calculate_source_version() or '0.0.0'
```

Thoughts?